polls/models.py

```diff
@@ -1,3 +1,54 @@
+from tabnanny import verbose
```

Comment:
Uhhh what is this and why did you add it? You didn't see the warnings from LGTM that this import is unused?

Reply:
Sorry, this came from VS code auto import, I have disabled it and will also remove this import in next commit

Reply:
@Programmer-yar Please review your PR diffs before asking for review. Thank you.

Reply:
Alright